src/StrawberryShake/CodeGeneration/src/CodeGeneration/Utilities/SchemaHelper.cs

```diff
@@ -72,6 +72,10 @@ public static class SchemaHelper
                                 scalar.Name.Value,
                                 scalar.Description?.Value));
                         }
+                        else if (scalar.Name.Value == ScalarNames.Any)
+                        {
+                            builder.AddType(new AnyType());
```

Comment:
@michaelstaib  so the problem of this sits deep in the type system i think. 
We add all unkonw scalars as `AnyType`. The Typesystem somehow then thinks it knows the `AnyType` already when there is another scalar defined. I tried to find a way to fix this in the core, but i feel like i am running in circles and looking in the wrong places. 
Anyway, adding the AnyScalar explicitly fixes this particular issue. But this is something we have to keep in mind for stitching as well